### PR TITLE
update(docs): added -t flag on docker run command

### PR DESCRIPTION
**Pull request by joaorufi**
Signed-off-by: joaorufi <joao.rufino@checkmarx.com>

**Proposed Changes**
- Added -t flag on docker run command

I submit this contribution under the Apache-2.0 license.

### CODE REVIEW

File: docs/BADGE.md [hunk 1]
```diff
@@ -5,16 +5,18 @@ To create a badge and update it the user should run KICS in their CI.
 This can be done differently for each [integration we have](https://docs.kics.io/1.3.1/integrations/).
 
 Let's assume a manual CLI process using docker:
+
 ```bash
 #!/usr/bin/env bash
 echo "running KICS in the current dir and writing results.json"
-docker run -v $PWD:/path checkmarx/kics:latest scan -p "/path" -o "/path/"
+docker run -t -v $PWD:/path checkmarx/kics:latest scan -p "/path" -o "/path/"
```
Comment (rafaela-soares) [inline docs/BADGE.md:12]: Maybe we can also update in `docs/integrations_terraformer.md`, `docs/integrations_travisci.md`, `docs/integrations_aws_codebuild.md`, `docs/integrations_bamboo.md`, `docs/integrations_teamcity.md`, `examples/teamcity/settings.kts`, and `examples/travis/.travis.yml`

**Review by rafaela-soares — CHANGES_REQUESTED** [review]

**Review by nunoocx — APPROVED** [review]
LGTM. The integrations docs we can skip. It is optional anyway :)

File: docs/BADGE.md (hunk 1 above)
Comment (nunoocx) [reply docs/BADGE.md:12]: As discussed, we can skip this addition to the integration docs. It is an optional parameter.

**Review by rafaela-soares — APPROVED** [review]
LGTM 🚀

**Pull request merged**